CANTHERM.egg-info/PKG-INFO

```diff
@@ -0,0 +1,42 @@
+Metadata-Version: 1.2
```

Comment:
Can you add `CANTHERM.egg-info` to `.gitignore` so the files don't show up in the repo? (And remove these files with `git rm -r CANTHERM.egg-info`)

Reply:
*egg-info/ was already in there but I'll see if adding it explicitly helps!

Reply:
Maybe it didn't catch up, but thanks for updating it!